buildozer/def.bzl

```diff
@@ -80,6 +83,14 @@ _buildozer = rule(
         "error_on_no_changes": attr.bool(
             doc = "Exit with 3 on success, when no changes were made",
         ),
+        "format_on_write": attr.bool(
```

Comment:
I personally don't need nor want the differentiation between `-buildifier=''` and `-buildifier=/some/path/to/buildifier/binary`, as I always use `@com_github_bazelbuild_buildtools//buildifier:buildifier`. I'm not sure offhand of a better way to handle the three-way conditional that the `-buildifier` option can take without a second attribute:

* unset (don't format)
* set but with an empty value, or no value (format using the internal formatter)
* set with a value to an executable

Very open to ideas/input here.